graph/src/blockchain/block_stream.rs

```diff
@@ -165,7 +165,7 @@ pub trait FirehoseMapper<C: Blockchain>: Send + Sync {
 #[derive(Error, Debug)]
 pub enum FirehoseError {
     /// We were unable to decode the received block payload into the chain specific Block struct (e.g. chain_ethereum::pb::Block)
-    #[error("received gRPC block payload cannot be decoded")]
+    #[error("received gRPC block payload cannot be decoded: {0}")]
     DecodingError(#[from] prost::DecodeError),
```

Comment:
Where is this being used? I can't find on the codebase when looking for `DecodingError`. I'm curious what will be passed on `{0}` 👀

Reply:
Deriving `thiserror::Error` allows using (unnamed) fields in the generated `Display` impl.
In this case `{0}` would be replaced by the `prost::DecodeError` `fmt` output (from its `Display` impl).
Some examples in `thiserror` docs: https://docs.rs/thiserror/latest/thiserror/

Reply:
I understand `thiserror` library usage, I mean in our codebase where this variant is used with a parameter.